.github/workflows/pr.yml

```diff
@@ -0,0 +1,72 @@
+name: Clojure CI
+
+on:
+  pull_request:
+
+env:
+  LOG_DIR: ~/log
+  BIN_DIR: ~/bin
+  LEIN_ROOT: 1
+  CTIA_MAJOR_VERSION: 1
+  COMPOSE_URI: https://github.com/docker/compose/releases/download/1.9.0
+  COMPOSE_BIN: ~/bin/docker-compose-1-9-0
+  COMPOSE_LOG: ~/log/docker-compose.log
+  CTIA_STORE_ES_DEFAULT_HOST: 127.0.0.1
+  CTIA_STORE_ES_DEFAULT_INDEXNAME: elasticsearch
+  CTIA_STORE_ES_DEFAULT_CLUSTERNAME: elasticsearch
+  CTIA_HOOK_ES_HOST: 127.0.0.1
+  CTIA_HOOK_ES_PORT: 9200
+  CTIA_HOOK_ES_INDEXNAME: elasticsearch
+  CTIA_HOOK_REDIS_URI: redis://127.0.0.1:6379
+  CTIA_NSPLITS: 20 # number of matrix builds
+  SHELLCHECK_VERSION: "v0.7.0" # "stable" or "v0.4.7", or "latest"
+
+jobs:
+  ## TODO fan in and deploy
+  # deploy:
+  #   needs: test
+  test:
+    runs-on: ubuntu-latest
```

Comment:
`-latest` can break for unrelated reasons, generally I'd go for a fixed version that always can be reproduced

Reply:
Pinned to 18.04.